R/Gargle-class.R

```diff
@@ -22,10 +32,10 @@ gargle2.0_token <- function(email = NULL,
                             scope = NULL,
                             user_params = NULL,
                             type = NULL,
-                            use_oob = getOption("httr_oob_default"),
+                            use_oob = getOption("gargle.oob_default"),
                             ## params end
                             credentials = NULL,
-                            cache = getOption("httr_oauth_cache"), ...) {
+                            cache = if (is.null(credentials)) getOption("gargle.oauth_cache") else FALSE, ...) {
```

Comment:
It is helpful to *be able* to cache tokens that hold directly-provided credentials, though you often don't want to. This is entirely about testing.